index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>testing</title>
+    <link rel="stylesheet" type="text/css" href="/Users/Yvonne/wdi/projects/tic_tac_toe/style2.css"/>
```

Comment:
Here's the problem! You're using an absolute path, instead of a relative path. This will only work on someone's computer if they have a Users folder, then a Yvonne folder, then a wdi folder, etc.